modifications/microshift-rebase

```diff
@@ -15,28 +15,23 @@ fail() {
     exit 1
 }
 
+# If MICROSHIFT_NO_REBASE is set and non-zero, stop execution here
 if [ -n "$MICROSHIFT_NO_REBASE" ] && [ "$MICROSHIFT_NO_REBASE" -ne 0 ] ; then
     echo "MICROSHIFT_NO_REBASE is defined. Rebase will not be run."
     exit 0
 fi
 
-# FIXME: rebase.sh requires Go yq (https://github.com/mikefarah/yq) instead of Python yq (https://github.com/kislyuk/yq),
-# which is used in the rest of the pipeline.
-# We install Go yq to /opt/yq-go/bin/ on buildvm to avoid conflict.
-# Use Go 1.19.5 as the version installed on buildvm (1.17) is no longer supported.
-# In the future we might have to run rebase.sh inside of a container to use specific versions of dependencies (including golang).
-export PATH=/opt/go-1.19.5/bin:/opt/yq-go/bin:$PATH
 
-if [ -n "$RELEASE_NAME" ] && [ -n "$LVMS_OPERATOR_BUNDLE" ]; then
-    # rebase against a named release
-    ./scripts/auto-rebase/rebase.sh to "$RELEASE_REPO":"$RELEASE_NAME"-x86_64 "$RELEASE_REPO":"$RELEASE_NAME"-aarch64 "$LVMS_OPERATOR_BUNDLE"
-    exit 0
-fi
-
-if [ -n "$MICROSHIFT_PAYLOAD_X86_64" ] && [ -n "$MICROSHIFT_PAYLOAD_AARCH64" ] && [ -n "$LVMS_OPERATOR_BUNDLE" ]; then
-    # rebase against specified release payloads
-    ./scripts/auto-rebase/rebase.sh to "$MICROSHIFT_PAYLOAD_X86_64" "$MICROSHIFT_PAYLOAD_AARCH64" "$LVMS_OPERATOR_BUNDLE"
-    exit 0
-fi
+PODMAN_BUILD_CMD="podman build -t microshift-rebase ."
```

Comment:
The image name `microshift-rebase` cannot be hardocoded here, otherwise there will be a race condition when multiple builds are running.

Reply:
Actually when the modification script is called by Doozer, the current directory is the local distgit repo of microshift instead of the `modification/` directory of ocp-build-data.

Reply:
So the current directory `.` shouldn't be used here.

You can use script like this to get the path to the modification directory:

```sh
DIR=$(dirname $(readlink -f $0))
```